dvc/scm/git.py

```diff
@@ -173,3 +173,10 @@ def _install_hook(self, name, cmd):
     def install(self):
         self._install_hook("post-checkout", "checkout")
         self._install_hook("pre-commit", "status")
+
+    def belongs_to_scm(self, path):
+        return os.path.basename(
```

Comment:
This is formatted (by black) that it is hard to read. Could you re-organize it, please? There is too much logic in this single line(kinda) statement.